.vib/cluster-autoscaler/goss/cluster-autoscaler.yaml

```
@@ -0,0 +1,6 @@
+command:
+  check-cluster-autoscaler-help:
+    exec: cluster-autoscaler --help || true
+    exit-status: 0
```

I would suggest not omitting the original exit code using `|| true` as this may hide other malfunction error codes.

The exit code `2` means 'invalid usage', which in this case is correct because the 'cluster-autoscaler' does not support the `--help` flag at the moment.

```suggestion
    exec: cluster-autoscaler --help
    exit-status: 2
```

Cc. @FraPazGal @joancafom